routes/teams.js

```diff
@@ -39,9 +40,9 @@ var returnItem = function (item, res){
   res.json(item);
 }
 
-var createQuiz = function(rb_team_id, league, res, callback){
+var createQuiz = function(rb_team_id, league, api_team_id, res, callback){
```

Comment:
bc of line 23 in routes/quiz.js.   If we only have the quiz id..then I can call mongo just once. 
Can easily tweak it to call mongo again for that encrypted team id if needed. 

What do you think?